modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,320 @@
+
+== Deploying a Sync Gateway Cluster
+
+The Sync Gateway nodes in a cluster have a homogeneous configuration with the exception of import node.
+
+Importer node:: Under xref:shared-bucket-access.adoc[convergence/shared bucket access], it is recommended that one Sync Gateway node in a cluster be configured for handling document import processing.
+For high availability, you can configure more than one Sync Gateway node in your cluster to be the import node, although it is strongly discouraged for multiple Sync Gateway nodes in the cluster to be configured for import processing.
+The configuration of the Sync Gateway import node is slightly different than the "regular" Sync Gateway nodes (see xref:config-properties.adoc#databases-foo_db-import_docs[databases.$db.import_docs]).
+//Replicator node:: if you are using inter-cluster replication using sg-replicate then there will be one designated replicator node whose configuration is different than the rest of the nodes.
+
+The following sections cover the steps to deploy regular or import Sync Gateway nodes.
+// or a single replicator node.
+
+You will have to repeat the steps in this section once for the "regular" nodes and once for the "import" node. The difference is the  Sync Gateway configuration file and the Kubernetes Deployment Controller config file.
+
+=== Configuration file
+
+In this section, you will create the Sync Gateway configuration file and store it in a secret location.
+
+. Open the configuration file corresponding to your deployment.
++
+|===
+| Regular Node | Importer Node
+
+| link:{attachmentsdir}/kubernetes/sgw-config.json[sgw-config.json]
+| link:{attachmentsdir}/kubernetes/sgw-config-import.json[sgw-config-import.json]
+|===
+. Replace the `server` key with the addressable Couchbase Server hostname.
+The following example shows the configuration file for a regular node.
++
+[source,json]
+----
+{
+  "logging": {
+    "console": {
+      "log_file_path": "/var/tmp/sglogs",
+      "console": {
+        "log_level": "debug",
+        "log_keys": ["*"]
+      },
+      "error": {
+        "enabled": true,
+        "rotation": {
+          "max_size": 20,
+          "max_age": 180
+        }
+      },
+      "warn": {
+        "enabled": true,
+        "rotation": {
+          "max_size": 20,
+          "max_age": 90
+        }
+      },
+      "info": {
+        "enabled": false
+      },
+      "debug": {
+        "enabled": false
+      }
+    }
+  },
+  "databases": {
+    "db": {
+      "server": "http://cb-example-0000.cb-example:8091", // <1>
+      "bucket": "default",
+      "username": "admin", // <2>
+      "password": "password",
+      "users": { "GUEST": { "disabled": false, "admin_channels": ["*"] } },
+      "allow_conflicts": false,
+      "revs_limit": 20
+    }
+  }
+}
+----
+<1> The server key should point to the Couchbase Server cluster.
+This would typically be of the form `http://CB_SERVER_POD.CB_SERVER_SERVICE_NAME:8091`.
```

Comment:
@mattcarabine  Can you confirm that this is accurate representation of path to server pod. Is this service name or cluster name that follows pod name?

Reply:
@mattcarabine  Thats not very different from non-containerized env right? Even today, SGW is configured points to one of CBS nodes in cluster. But makes sense to clarify that. 
That said, the full form seems different than what was configured on SGW(http://cb-example-0000.cb-example:8091) . Let me verify with the URL that you provided above

Reply:
@mattcarabine $`CB_SERVER_POD.couchbase.$NAMESPACE.srv:8091` did not work (the SGW failed to connect to the bucket). I had to use `$CB_SERVER_POD.$CB_CLUSTER_NAME:8091`.  I used standard server side configuration. What did I miss?